guides/source/association_basics.md

```diff
@@ -2857,7 +2853,7 @@ end
 
 With this definition complete, our `Entry` delegator now provides the following methods:
 
-```ruby
+```
```

Comment:
Why isn't this ruby anymore?

Reply:
It's parseable Ruby in that every `Entry` is a thing, but `Entry#whatever` will just be a comment after that. I think by specifying ruby as the code block, it means this code must adhere to our style guidelines.

I've also considered rewriting this as a table, because I prefer that to code for readability, e.g. (patch later):

![Screenshot 2023-02-02 at 10 46 21](https://user-images.githubusercontent.com/277819/222856794-0e13cc83-fe81-4b55-b334-225700cf2e95.png)

Reply:
Table would be a good solution for this.

Reply:
Resolved in #47653